frontend/packages/noobaa-storage-plugin/src/constants/index.ts

```diff
@@ -4,3 +4,10 @@ export const BY_IOPS = 'I/O Operations';
 export const BY_LOGICAL_USAGE = 'Logical Used Capacity';
 export const BY_PHYSICAL_VS_LOGICAL_USAGE = 'Physical Vs Logical Usage';
 export const BY_EGRESS = 'Egress';
+
+export const USAGE_LABELS = {
```

Comment:
Just a tiny suggestion, can we name it something like CHART_LABELS

Reply:
Acknowledged...